composeApp/src/commonMain/kotlin/org/ooni/probe/domain/GetAutoRunSettings.kt

```diff
@@ -0,0 +1,29 @@
+package org.ooni.probe.domain
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import org.ooni.probe.data.models.AutoRunParameters
+import org.ooni.probe.data.models.SettingsKey
+
+class GetAutoRunSettings(
```

Comment:
Since we now have this class, we could use it directly in `ObserveAndConfigureAutoRun`, instead of having the duplicated code there.